_posts/2019-08-09-getting-started.md

```diff
@@ -123,13 +121,13 @@ Quickly check the files needed for GitHub Actions build:
 
 After the above steps, rename your repository to `<GH_USERNAME>.github.io` on GitHub.
 
-Now publish your Jekyll site by:
+Now publish your Jekyll site:
 
-1. Push any commit to remote to trigger the GitHub Actions workflow. Once the build is complete and successful, a new remote branch named `gh-pages` will appear to store the built site files.
+1. Browse to your repository on GitHub. Select the tab _Settings_, then click _Pages_ in the left navigation bar. Then, in the **Source** section (under _Build and deployment_), select [**GitHub Actions**][pages-workflow-src] from the dropdown menu.
 
-2. Browse to your repository on GitHub. Select the tab _Settings_, then click _Pages_ in the left navigation bar, and then in the section **Source** of _GitHub Pages_, select the `/(root)` directory of branch `gh-pages` as the [publishing source][pages-src]. Remember to click <kbd>Save</kbd> before leaving.
+2. Push any commit to remote to trigger the GitHub Actions workflow. In the _Actions_ tab of your repository, you should see the "Automatic build and deploy" workflow with at least one job running. Once the build is complete and successful, the site should be deployed automatically.
 
-    ![gh-pages-sources](/posts/20190809/gh-pages-sources.png){: width="1580" height="250" }
+    <!-- ![gh-pages-sources](/posts/20190809/gh-pages-sources.png){: width="1580" height="250" } -->
```

Comment:
I am not able to access the CDN here to change this image, so I just omitted this in my change for now.

Reply:
Thank you so much for your willingness to update the docs. The images are stored in another repository, I'll modify the URL later.